### PR TITLE
dfu-util: Added x86 build for Haiku R1B2

### PR DESCRIPTION
**Pull request by kenmays**
Supported Devices

    Openmoko Neo1973
    Openmoko Freerunner
    Leaflabs Maple
    OpenPCD and OpenPICC
    Qi Hardware ATUSB (use dfu-util 0.7)
    Osmocom SIMtrace
    Many devices using the DfuSe DFU extension from ST, for instance:
        Black Magic Probe
        DSO nano
        Spark Core / Particle Photon
        STM32F2/F3/F4 built-in bootloader
        Some STR750-based devices, e.g. USB2CAN

### CODE REVIEW

File: app-mobilephone/dfu-util/dfu_util-0.9.recipe [hunk 1]
```diff
@@ -8,27 +8,28 @@ been tested with the Openmoko Neo1973 and Freerunner and many other devices."
...
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	dfu_util = $portVersion
-	cmd:dfu_prefix
-	cmd:dfu_suffix
-	cmd:dfu_util
+	dfu_util$secondaryArchSuffix = $portVersion
+	cmd:dfu_prefix$secondaryArchSuffix
```
Comment (extrowerk) [inline app-mobilephone/dfu-util/dfu_util-0.9.recipe]: Why not commandSuffix?

File: app-mobilephone/dfu-util/dfu_util-0.9.recipe [hunk 2]
```diff
@@ -40,7 +48,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	CFLAGS=-D_BSD_SOURCE runConfigure ./configure
+	CFLAGS=-D_BSD_SOURCE runConfigure ./configure --bindir=$commandBinDir
```
Comment (extrowerk) [inline app-mobilephone/dfu-util/dfu_util-0.9.recipe:51]: Ommit should be added.

Comment (Begasus) [reply app-mobilephone/dfu-util/dfu_util-0.9.recipe:51]: Suggestion:
```diff
- 	CFLAGS=-D_BSD_SOURCE runConfigure ./configure
+ 	export CFLAGS="-D_BSD_SOURCE"
+ 	runConfigure --omit-dirs binDir ./configure \
+ 	 	--bindir=$commandBinDir
```

Comment (kenmays) [reply app-mobilephone/dfu-util/dfu_util-0.9.recipe]: Fixed

File: app-mobilephone/dfu-util/dfu_util-0.9.recipe (hunk 1 above)
Comment (kenmays) [reply app-mobilephone/dfu-util/dfu_util-0.9.recipe]: Fixed

**Pull request merged**